src/main/java/org/ccci/idm/rules/services/RoleManagerServiceUserManager.java

```diff
@@ -0,0 +1,185 @@
+package org.ccci.idm.rules.services;
+
+import com.google.common.collect.ImmutableMap;
+import org.ccci.idm.obj.RoleAssignment;
+import org.ccci.idm.user.Group;
+import org.ccci.idm.user.User;
+import org.ccci.idm.user.UserManager;
+import org.ccci.idm.user.ldaptive.dao.io.GroupValueTranscoder;
+import org.ccci.util.strings.Strings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+
+public class RoleManagerServiceUserManager implements RoleManagerService
+{
+    private Logger logger = LoggerFactory.getLogger(getClass());
+
+    private String attestorId;
+    private String roleBasePath;
+
+    private UserManager userManager;
+
+    private GroupValueTranscoder groupValueTranscoder;
+
+    private static final Map<String, String> RoleMap = ImmutableMap.of(
+            "roles:StaffOnlyConsumer", "cn=StaffOnlyConsumer,ou=Roles"
```

Comment:
It's resolved by `convertRoleNameToFullPath`. I'm not sure why the code was written this way originally (I did not write it), because it seems it could probably be set to full path at definition of `RoleMap`. But, I simply left the original design as is.